src/_posts/languages/python/2000-01-01-start.md

```diff
@@ -14,16 +14,33 @@ To ensure our deployment system considers your application as a Python
 application, a file `Pipfile`, `requirements.txt` or `setup.py` should be
 present at the root of your project, defining the dependencies of your app.
 
-## Versions
+## Python Versions
 
-## Supported Versions
+### Availability
 
-* `3.11.4` (default)
-* `3.10.12`
-* `3.9.17`
-* `3.8.17` (only for `scalingo-20` stack)
+The following versions of Python are available:
 
-### Recommended: pipenv
+| Python Version | `scalingo-20`   | `scalingo-22`   |
+| -------------: | --------------: | --------------: |
+| `3.12`         | up to `3.12.2`  | up to `3.12.2`  |
+| `3.11`         | up to `3.11.8`  | up to `3.11.8`  |
+| `3.10`         | up to `3.10.13` | up to `3.10.13` |
+| `3.9`          | up to `3.9.18`  | up to `3.9.18`  |
+| `3.8`          | up to `3.8.18`  | unsupported     |
+
+{% note %}
+Even though we still support them, we strongly advise against using deprecated
+versions of Python, such as Python `3.7.x`. If you are still using one of these
+deprecated versions, please consider migrating to a more recent version.
+{% endnote %}
+
+### Selecting a Version
+
+The default Python version on both `scalingo-20` and `scalingo-22` is the
+latest `3.12` version.\
```

Comment:
question What is this backslash about? :thinking:

Reply:
It creates a `<br />` a line-feed without starting a new paragraph.

Reply:
I'm not sure we want that kind of thing but OK

Reply:
I can remove it if you want, I don't care, really :smiley_cat: